helpers/deploymentConfig.ts

```diff
@@ -159,6 +161,13 @@ const preconfiguredAddresses = {
     GovernorBravo: ETHEREUM_MULTISIG,
     AccessControlManager: governanceEthereum.AccessControlManager.address,
   },
+  opbnbtestnet: {
+    VTreasury: "0x3370915301E8a6A6baAe6f461af703e2498409F3",
+    NormalTimelock: OPBNBTESTNET_MULTISIG,
+    FastTrackTimelock: OPBNBTESTNET_MULTISIG,
+    CriticalTimelock: OPBNBTESTNET_MULTISIG,
+    AccessControlManager: "0x049f77F7046266d27C3bC96376f53C17Ef09c986",
```

Comment:
This address should be available on the npm package (at least in the version `1.4.0-dev.9`)